guide/source/program_administrator/introduction.rst

```diff
@@ -87,7 +89,7 @@ will want to familiarize yourself with this group, as they can serve as an
 invaluable resource to assist in the implementation of the BPI data standards,
 facilitate changes in the standard, and answer questions that you may have
 about the standards. To contact the BPI WG-5 or to get involved, email them at
-Julie.Caracino@nyserda.ny.gov.
+Julie.Caracino@nyserda.ny.gov. [IT WOULD BE PREFERABLE TO INCLUDE A GENERIC EMAIL ADDRESS LIKE HPXML@BPI.ORG INSTEAD OF A PERSON'S EMAIL ADDRESS IN A GUIDE LIKE THIS. OTHERWISE, WE COULD HAVE THEM GO TO HOMEPERFORMANCE@ENERGYSTAR.GOV]
```

Comment:
Starting a chain so I can be involved…


From: Noel Merket [mailto:notifications@github.com]
Sent: Tuesday, January 20, 2015 10:16 AM
To: hpxmlwg/hpxml
Cc: Gavin Hastings
Subject: Re: [hpxml] Ely's Edits (#23)


In guide/source/program_administrator/introduction.rst<https://github.com/hpxmlwg/hpxml/pull/23#discussion-diff-23237756>:

> @@ -87,7 +89,7 @@ will want to familiarize yourself with this group, as they can serve as an

>  invaluable resource to assist in the implementation of the BPI data standards,

>  facilitate changes in the standard, and answer questions that you may have

>  about the standards. To contact the BPI WG-5 or to get involved, email them at

> -Julie.Caracino@nyserda.ny.gov<mailto:-Julie.Caracino@nyserda.ny.gov>.

> +Julie.Caracino@nyserda.ny.gov<mailto:+Julie.Caracino@nyserda.ny.gov>. [IT WOULD BE PREFERABLE TO INCLUDE A GENERIC EMAIL ADDRESS LIKE HPXML@BPI.ORG<mailto:HPXML@BPI.ORG> INSTEAD OF A PERSON'S EMAIL ADDRESS IN A GUIDE LIKE THIS. OTHERWISE, WE COULD HAVE THEM GO TO HOMEPERFORMANCE@ENERGYSTAR.GOV<mailto:HOMEPERFORMANCE@ENERGYSTAR.GOV>]

Nope. I have no details about that. I'll follow up with him.

—
Reply to this email directly or view it on GitHub<https://github.com/hpxmlwg/hpxml/pull/23/files#r23237756>.